api/eid_service/views.py

```diff
@@ -110,4 +113,4 @@ def error(request):
     The user agent is redirected to this view if the authentication session between eID-Client
     and eID-Server fails.
     """
-    return HttpResponseRedirect('https://' + request.get_host() + '/api/openid/login')
+    return HttpResponseRedirect(reverse(eid_openid_views.open_id_login))
```

Comment:
Right now, this redirects to an url that reacts with a BadRequest (I suppose that happens, because the AuthorizationRequest is missing?). Am I right that adding the initial authorization request to this redirect would enable the user to start a whole new authorization attempt?